### PR TITLE
add new package "freifunk-berlin-lua-tools"

### PR DESCRIPTION
**Pull request by SvenRoederer**
Move some generic code from the assistent into this new common package.
This way the functions can be used by other code as well.

### CODE REVIEW

**Comment by SvenRoederer** [comment]
I have not checked if the wizard still works after this change, but feedback on the overall idea is very welcome.

File: addons/freifunk-berlin-lua-tools/Makefile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=Freifunk Berlin lua tools
+LUCI_DEPENDS:=luci-lib-base
```
Comment (PolynomialDivision) [inline addons/freifunk-berlin-lua-tools/Makefile:5]: +luci-lib-base ?

Comment (SvenRoederer) [reply addons/freifunk-berlin-lua-tools/Makefile:5]: > +luci-lib-base ?

https://github.com/openwrt/luci/pull/2817

Comment (SvenRoederer) [reply addons/freifunk-berlin-lua-tools/Makefile:5]: got merged upstream: https://github.com/openwrt/luci/pull/2817#issuecomment-660873641

**Pull request merged**